packages/gatsby/src/schema/node-model.js

```diff
@@ -977,7 +986,7 @@ const addRootNodeToInlineObject = (
   }
 }
 
-const saveResolvedNodes = async (typeName, resolvedNodes) => {
+const saveResolvedNodes = (typeName, resolvedNodes) => {
```

Comment:
random change I noticed while I was around this code - this is not `async` function so no need for that keyword